Tests/CommandsTests/BuildCommandTests.swift

```diff
@@ -1760,6 +1760,45 @@ struct BuildCommandTestCases {
             data.config == .release
         }
     }
+
+    @Test(
+        .requireHostOS(.macOS),
+        .tags(
+            .Feature.CommandLineArguments.BuildSystem,
+            .Feature.CommandLineArguments.Configuration,
+        ),
+        arguments: getBuildData(for: [.swiftbuild]),
+    )
+    func trivialPackageResources(
+        data: BuildData,
+    ) async throws {
+        let buildSystem = data.buildSystem
+        try await fixture(name: "Miscellaneous/TIF") { fixturePath in
+            let result = try await build(
+                [],
+                packagePath: fixturePath,
+                configuration: data.config,
+                cleanAfterward: false,
+                buildSystem: buildSystem,
+            )
+            try #require(result.binContents.contains("TIF_TIF.bundle"))
+            let contentsDir = result.binPath.appending("TIF_TIF.bundle", "Contents")
+            let bundleResourceDir = contentsDir.appending("Resources")
```

Comment:
Question: is there value is validating this path exists so we can have "better" message in the event it does not?